packages/forms-web-app/src/views/layouts/vertical-tabs.njk

```diff
@@ -16,7 +16,7 @@
     </li>
     <li class="side-menu govuk-body-s {% if 'about-the-application' in current %} active {% endif %}">
       {% if not 'about-the-application' in current %}
-        <a href="/examination/about-the-application/1">Project application documents</a>
+        <a href="/examination/about-the-application/{{ caseRef }}/1">Project application documents</a>
```

Comment:
Again href looks inconsistent with route change.

Reply:
all routes/links updated to have /examination/{{case_ref}}